### PR TITLE
OKTA-405245 moving Social Idp settings content

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Moved "Social Identity Provider settings" page (/docs/reference/social-settings) to an H2 section within "Add an external Identity Provider" > "Create an Identity Provider in Okta" page (/docs/guides/add-an-external-idp/-/configure-idp-in-okta). Updated conductor.yml to redirect URL to new location.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-405245](https://oktainc.atlassian.net/browse/OKTA-405245)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/social-login/index.md [hunk 1]
```diff
@@ -70,4 +70,4 @@ To set up social login, configure the following:
 2. An Identity Provider in Okta
 3. An OpenID Connect Application in Okta
 
-Every Identity Provider in Okta is linked to an Application, and every time a user signs in with a social Identity Provider for the first time, an Application User is created for them. The Application User represents the external user at the social Identity Provider and can be used to map attributes to the Okta User. See [Social Identity Provider settings](/docs/reference/social-settings/) for more information about how to configure this behavior.
+Every Identity Provider in Okta is linked to an Application, and every time a user signs in with a social Identity Provider for the first time, an Application User is created for them. The Application User represents the external user at the social Identity Provider and can be used to map attributes to the Okta User. See [Social Identity Provider settings](/docs/guides/add-an-external-idp/-/configure-idp-in-okta/#social-identity-provider-settings) for more information about how to configure this behavior.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/concepts/social-login/index.md:73]:
```suggestion
Every Identity Provider in Okta is linked to an Application, and every time a user signs in with a social Identity Provider for the first time, an Application User is created for them. The Application User represents the external user at the social Identity Provider and can be used to map attributes to the Okta User. See [Social Identity Provider settings](/docs/guides/add-an-external-idp/-/configure-idp-in-okta/#social-identity-provider-settings) for more information on how to configure this behavior.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/apple/appidpinokta.md [hunk 2]
```diff
@@ -9,4 +9,4 @@ In the **General Settings** section:
 
     By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory. For Just In Time (JIT) provisioning, Okta also requires the `firstName` and `lastName` attributes for a user. Include the `name` scope if the Identity Provider needs to support JIT.
 
-> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](/docs/reference/social-settings/).
\ No newline at end of file
+> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](#social-identity-provider-settings).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/apple/appidpinokta.md:12]:
```suggestion
> **Note:** See [Social Identity Provider Settings](#social-identity-provider-settings) for more information on these settings and the **Advanced Settings**.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/facebook/appidpinokta.md [hunk 3]
```diff
@@ -5,4 +5,4 @@
 
     By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
 
-> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](/docs/reference/social-settings/).
+> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](#social-identity-provider-settings).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/facebook/appidpinokta.md:8]:
```suggestion
> **Note:** See [Social Identity Provider Settings](#social-identity-provider-settings) for more information on these settings and the **Advanced Settings**.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/google/appidpinokta.md [hunk 4]
```diff
@@ -5,4 +5,4 @@
 
     By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
 
-> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](/docs/reference/social-settings/).
+> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](#social-identity-provider-settings).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/google/appidpinokta.md:8]:
```suggestion
> **Note:** See [Social Identity Provider Settings](#social-identity-provider-settings) for more information on these settings and the **Advanced Settings**.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md [hunk 5]
```diff
@@ -19,7 +19,40 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
-### Attribute Mapping
+## Social Identity Provider settings
+
+When you are setting up your social Identity Provider (IdP) in Okta, there are a number of settings that allow you to finely control the social sign-in behavior. While the provider-specific instructions show one possible configuration, this section explains each of these in more detail so that you can choose the right configuration for your use case.
+
+### Authentication settings
+
+**IdP Username:** The expression (written in the Okta Expression Language) that is used to convert an IdP attribute to the Application User's `username`. This IdP username is used for matching an Application User to an Okta User through the `oidc_idp` profile.
+
+You can enter an expression to reformat the value, if desired. For example, if the social username is `john.doe@mycompany.com`, you could specify the replacement of `mycompany` with `endpointA.mycompany` to make the transformed username `john.doe@endpointA.mycompany.com`. See [Okta Expression Language](/docs/reference/okta-expression-language/).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md:30]:
```suggestion
You can enter an expression to reformat the value. For example, if the social username is `john.doe@mycompany.com`, you could specify the replacement of `mycompany` with `endpointA.mycompany` to make the transformed username `john.doe@endpointA.mycompany.com`. See [Okta Expression Language](/docs/reference/okta-expression-language/).
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md [hunk 6]
```diff
@@ -19,7 +19,40 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
+
+**Match against &mdash;** The Okta user property against which the IdP username is compared to determine if an account link needs to be established. If an existing account link is found, no comparison is performed.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md:32]:
```suggestion
**Match against &mdash;** The Okta user property that the IdP username is compared against in order to determine if an account link needs to be established. If an existing account link is found, no comparison is performed.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/linkedin/appidpinokta.md [hunk 7]
```diff
@@ -5,4 +5,4 @@
 
     By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
 
-> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](/docs/reference/social-settings/).
+> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](#social-identity-provider-settings).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/linkedin/appidpinokta.md:8]:
```suggestion
> **Note:** See [Social Identity Provider Settings](#social-identity-provider-settings) for more information on these settings and the **Advanced Settings**.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/microsoft/appidpinokta.md [hunk 8]
```diff
@@ -5,4 +5,4 @@
 
     By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
 
-> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](/docs/reference/social-settings/).
+> **Note:** For more information about these settings as well as the **Advanced Settings**, see [Social Identity Provider Settings](#social-identity-provider-settings).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/microsoft/appidpinokta.md:8]:
```suggestion
> **Note:** See [Social Identity Provider Settings](#social-identity-provider-settings) for more information on these settings and the **Advanced Settings**.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md [hunk 9]
```diff
@@ -18,8 +18,8 @@ We recommend that if the external IdP requires information from Okta for setup b
 
 * **IdP Issuer URI** &mdash; the issuer. The Identity Provider provides this value.
 
-* **IdP Single Sign-On URL** &mdash; the sign-on URL from the Identity Provider. If you sign the authN request by selecting the **Request Signature** option, but don't specify a destination in the **Destination** field (see [Advanced Settings](/docs/reference/social-settings/)), Okta automatically sends the authN request to the Identity Provider Single Sign-On URL.
+* **IdP Single Sign-On URL** &mdash; the sign-on URL from the Identity Provider. If you sign the authN request by selecting the **Request Signature** option, but don't specify a destination in the **Destination** field (see [Advanced Settings](#social-identity-provider-settings)), Okta automatically sends the authN request to the Identity Provider Single Sign-On URL.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md:21]:
```suggestion
* **IdP Single Sign-On URL** &mdash; The sign-on URL from the Identity Provider. If you sign the authN request by selecting the **Request Signature** option but don't specify a destination in the **Destination** field (see [Advanced Settings](#social-identity-provider-settings)), Okta automatically sends the authN request to the Identity Provider Single Sign-On URL.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md [hunk 10]
```diff
@@ -18,8 +18,8 @@ We recommend that if the external IdP requires information from Okta for setup b
...
 
 * **IdP Signature Certificate** &mdash; click **Browse files** to upload the certificate from the Identity Provider used to sign the assertion.
 
-> **Note:** For more information about **Advanced Settings**, see [Social Identity Provider Settings](/docs/reference/social-settings/).
+> **Note:** For more information about **Advanced Settings**, see [Social Identity Provider Settings](#social-identity-provider-settings).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md:25]:
```suggestion
> **Note:** See [Social Identity Provider Settings](#social-identity-provider-settings) for more information on **Advanced Settings**.
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/configure-oidc-native-apps/index.md [hunk 11]
```diff
@@ -10,7 +10,7 @@ Within the same org, you need to set up two Native OpenID Connect (OIDC) client
 1. Give the app integration a name, and then enter `com.first.sample:/callback` in the **Sign-in redirect URIs** box for the first app.
     > **Note:** When you create the second app, enter `com.second.sample:/callback`.
 1. Ensure that **Authorization Code** and **Refresh Token** are selected in the **Grant Type Allowed** section.
-1. Assign the group that you want (if you set [Group Assignments](/docs/reference/social-settings/) for your app) or leave the **Everyone** default. For instructions on how to assign the app integration to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
+1. Assign the group that you want (if you set Group Assignments for your app) or leave the **Everyone** default. For instructions on how to assign the app integration to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/configure-oidc-native-apps/index.md:13]:
```suggestion
1. Assign the group that you want (if you set Group Assignments for your app) or leave the **Everyone** default. See the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation for instructions on how to assign the app integration to individual users and groups.```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md [hunk 12]
```diff
@@ -15,7 +15,7 @@ Before you can sign a user in, you need to create an Okta app integration that r
...
 1. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
-1. Assign the group that you want (if you set [Group Assignments](/docs/reference/social-settings/) for your app) or leave the **Everyone** default. For instructions on how to assign the app integration to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
+1. Assign the group that you want (if you set Group Assignments for your app) or leave the **Everyone** default. For instructions on how to assign the app integration to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md:18]:
```suggestion
1. Assign the group that you want (if you set Group Assignments for your app) or leave the **Everyone** default. See the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation for instructions on how to assign the app integration to individual users and groups.
```

**Review by christinechoy-okta — COMMENTED** [review]
Just a few comments...

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md (hunk 9 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md:21]: I'm going to leave the lower case to be consistent with the rest of the bullet descriptions.

**Review by christinechoy-okta — APPROVED** [review]

**Pull request merged**